docs/SUMMARY.md

```diff
@@ -135,11 +128,18 @@
         * [Connection](database/connection.md)
         * [Connection pool](database/connection_pool.md)
         * [Transactions](database/transactions.md)
-* Tutorials
-    * [Basics](tutorials/basics/README.md)
+    * [Coding style](conventions/coding_style.md)
+* [Tutorials](tutorials/README.md)
+    * [Getting started](getting_started/README.md)
+        * [An HTTP Server](getting_started/http_server.md)
+        * [A Command Line Application](getting_started/cli.md)
```

Comment:
I'm also not convinced that *getting started* properly classifies as a tutorial.

Reply:
How would you call it? A _guide_? To me _tutorial_ fits better. But the distinction isn't that clear...

Reply:
It might not fit the strict definition of what we consider a tutorial, but it fits very well in this place, so let's leave it like that.